robottelo/common/ssh.py

```diff
@@ -153,26 +153,20 @@ def command(cmd, hostname=None, output_format=None, timeout=None):
 
     if stdout:
         stdout = stdout.decode('utf-8')
-        logger.debug("<<<\n%s", stdout)
+        logger.debug('<<<\n%s', stdout)
+    if stderr:
+        stderr = stderr.decode('utf-8')
```

Comment:
Looking at the old code, `stderr` was a list of error messages and we were joining them before displaying it (`if errorcode != 0`). Is this necessary here?